requirements.txt

```diff
@@ -1,16 +1,12 @@
-black>=24.10.0
```

Comment:
do we have no over numpy versioning?

Reply:
Not that I'm aware of but I'll leave this comment unresolved until I test locally. 

I think that "over-pin" or whatever you want to call it was in place because of an unresolved issue with an ancient Lightning not playing nicely with very new Numpy.

Reply:
If you wouldn't mind, yeah those tests would be appreciated. Numpy versioning is just bane of my existence so would rather pin that down now before they or the PTL people decide to torment us

Reply:
This is resolved; I tested it with the oldest and newest Numpy (> than the old pin) and it works.